internal/crypto/crypto.go

```diff
@@ -0,0 +1,70 @@
+package crypto
```

Comment:
Used in address_test.go. Sits under `internal/` to prevent it from being used outside of this library.